shared/src/androidMain/kotlin/com/habitrpg/shared/habitica/MultiplatformRealmLibrary.kt

```diff
@@ -0,0 +1,6 @@
+package com.habitrpg.shared.habitica
+
+import io.realm.annotations.RealmModule
+
+@RealmModule(library = true, allClasses = true)
+actual open class MultiplatformRealmLibrary
```

Comment:
Required to introduce the shared library as a source of realm models, compilation will fail otherwise. See the initialization process for the usage.

Reply:
Habitica/src/main/java/com/habitrpg/android/habitica/HabiticaBaseApplication.kt:112

Reply:
I haven't tested it out yet, but it appears that iOS is able to handle this. If I'm correct from a brief glance https://realm.io/docs/swift/latest/#configuring-a-realm this seems to indicate all you need to do is load both the models. It looks like the library configuration is required for android only